app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,21 @@
+
+<% if obj.errors.any? %>
+  <div class="row errors">
+    <div class="col-sm-10 mx-auto">
+      <div class="card arlert alert-danger p-2 mb-3">
```

Comment:
:bulb: [OPTIONAL] There shouldn't be any logic(if) in the views. You should move them into the helpers.

**Resources** 
- [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
- [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
- [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

Reply:
:pray: Thanks so much Kubilay Caglayan. Your suggestions are helpful, :ok_hand: . I will definitely apply them in my projects. With great courtesy,  keep up the good work. :100: :thumbsup: